appveyor.yml

```diff
@@ -5,7 +5,6 @@ init:
 install:
   - SET PATH=C:\Ruby21-x64\bin;%PATH%
   - SET LOG_SPEC_ORDER=true
-  - gem install bundler --quiet --no-ri --no-rdoc
```

Comment:
If you're using AppVeyor's pre-installed Ruby versions (1.9.3-2.24) they all have Bundler 1.11.2 pre-installed.